bears/python/RadonBear.py

```diff
@@ -18,39 +19,56 @@ class RadonBear(LocalBear):
     CAN_DETECT = {'Complexity'}
 
     def run(self, filename, file,
-            radon_ranks_info: typed_list(str) = (),
-            radon_ranks_normal: typed_list(str) = ('C', 'D'),
-            radon_ranks_major: typed_list(str) = ('E', 'F'),
-            ):
+            cyclomatic_complexity: int=-1,
```

Comment:
use `None` for no setting. (and you'll need to adjust the setting type annotation to allow that.